### PR TITLE
Added documents for commandline arguments

### PR DESCRIPTION
**Pull request by shinglyu**
An initial draft for #10878

---

This change is [Reviewable](https://reviewable.io/reviews/servo/servo/10914)

### CODE REVIEW

File: docs/COMMAND_LINE_ARGS.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+Command Line Arguments
+========================
+# General
+
+You can see available commands with
```
Comment (aneeshusa) [inline docs/COMMAND_LINE_ARGS.md]: style nit: colon at the end of the line

File: docs/COMMAND_LINE_ARGS.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+# Run
+## Enable Experimental Features
+Use `--pref` to enable experimental features like experimental DOM API, JavaScript API and CSS properties.
+
+e.g. To enable `flex` and `flex-direction` css properties:
+```
+./mach run -d -- --pref layout.flex.enabled --pref layout.flex-direction.enabled ...
+```
+
+You can find all the available preferences at [resources/perfs.json](http://mxr.mozilla.org/servo/source/resources/prefs.json).
```
Comment (aneeshusa) [inline docs/COMMAND_LINE_ARGS.md]: spelling:`resources/prefs.json`

File: docs/COMMAND_LINE_ARGS.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+# Debugging
+## Remote Debugging
+Use `--devtools 6000` to start the devtools server on port 6000.
+
+e.g.
+```
+./mach run -d --devtools 6000 https://servo.org
+```
+
+To connect to the server, follow [this guide](https://developer.mozilla.org/en-US/docs/Tools/Remote_Debugging/Debugging_Firefox_Desktop#Connect)
```
Comment (aneeshusa) [inline docs/COMMAND_LINE_ARGS.md]: style nit: period at the end of the line

File: docs/COMMAND_LINE_ARGS.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+Command Line Arguments
+========================
+# General
+
+You can see available commands with:
+```
+./mach -h
+./mach <sub-comamnd> -h
```
Comment (aneeshusa) [inline docs/COMMAND_LINE_ARGS.md]: spelling: `sub-command`

**Comment by aneeshusa** [comment]
Thanks for writing this up!

@bors-servo r+

**Comment by ConnorGBrewster** [comment]
@bors-servo retry
 * #10924

**Comment by ConnorGBrewster** [comment]
@bors-servo retry
 * #10924

**Pull request merged**